tests/test_fluent_version.py

```diff
@@ -89,3 +97,21 @@ def test_eq():
 def test_str_output():
     assert str(FluentVersion.v232) == "Fluent version 2023 R2"
     assert str(FluentVersion.v251) == "Fluent version 2025 R1"
+
+
+def test_fluent_version_set():
```

Comment:
@mkundu1 I was thinking of a smaller set of functions that could be applied as compound functions but I prefer what you have done: when one sees this set of available functions, it is immediately clear how they should be used.